### PR TITLE
[17206] Restore participant domainID tag in XML profile

### PR DESCRIPTION
**Pull request by JesusPoderoso**
- Restore `<domainId>` tag in documentation
- Restore tag in ALL documentation XMLs

Rebased #455 

**This pull request should be merged right after #455 gets merged**

### CODE REVIEW

**Comment by JesusPoderoso** [comment]
@richiprosima Please test this

File: docs/fastdds/xml_configuration/domainparticipant.rst [hunk 1]
```diff
@@ -36,7 +36,7 @@ The ``<participant>`` element has two attributes defined: ``profile_name`` and `
        DomainParticipant's creation.
      - Optional
 
-.. _domainparticipantconfig:
+.. _domainparticipantconfig:
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/xml_configuration/domainparticipant.rst:39]: Please, remove trailing whitespace. CI should have complained but it failed downloading the repository. I have relaunched the job.

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

File: docs/fastdds/xml_configuration/domainparticipant.rst [hunk 2]
```diff
@@ -41,8 +41,26 @@ The ``<participant>`` element has two attributes defined: ``profile_name`` and `
 DomainParticipant configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``<participant>`` element has as child element ``<rtps>`` under which all the DomainParticipant configuration
-options belong.
+The ``<participant>`` element has two child elements: ``<domain_id>`` and ``<rtps>``.
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/xml_configuration/domainparticipant.rst:44]: If `<domain_id>` is not supported, it should be fixed here and refer only to `<domainId>` which is the supported one

File: code/XMLTester.xml [hunk 3]
```diff
@@ -955,6 +955,7 @@
 </participant>
 
 <participant profile_name="part_builtin_example">
+<domainId>4</domainId> <!-- uint32 -->
```
Comment (JLBuenoLopez-eProsima) [inline code/XMLTester.xml:958]: #435 removed the `<domainId>` tag from the `domainparticipant_profile_name` snippet and not from the `part_builtin_example` profile. Have you checked that the `<domainId>` tag is being shown in the snippet in the XML section?

Also, the tag is not indented.

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]

File: code/XMLTester.xml (hunk 3 above)
Comment (JesusPoderoso) [reply code/XMLTester.xml]: You were right. Doc built and tested locally successfully

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM!

**Review by JLBuenoLopez-eProsima — APPROVED** [review]

**Pull request merged**